File: app/src/main/java/br/com/dalcim/picpay/PicPayApplication.java
package br.com.dalcim.picpay;

import android.app.Application;

import uk.co.chrisjenx.calligraphy.CalligraphyConfig;

public class PicPayApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("fonts/Roboto-Light.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        );
    }
}

File: picpay/app/src/main/java/br/com/picpay/picpay/PicpayApplication.kt
package br.com.picpay.picpay

import android.app.Application
import br.com.picpay.picpay.di.component.ApplicationComponent
import br.com.picpay.picpay.di.component.DaggerApplicationComponent
import br.com.picpay.picpay.di.module.NetworkModule

class PicpayApplication: Application() {
    companion object {
        //platformStatic allow access it from java code
        @JvmStatic
        lateinit var graph: ApplicationComponent
    }

    override fun onCreate() {
        super.onCreate()

        graph = DaggerApplicationComponent
            .builder()
            .networkModule(NetworkModule(this))
            .build()

        graph.inject(this)
    }
}